nujo/utils/data/data_loader.py

```diff
@@ -0,0 +1,25 @@
+from os import mkdir
+from os.path import exists, expanduser
+import requests
+
+
+class DataLoader:
+    def __init__(self, name):
+        self._name = name
+        self._link = 'https://archive.ics.uci.edu/ml/machine-learning-databases/{0}/{0}.data'.format(
+            name)
+
+    def download(self):
```

Comment:
@ManiacMaxo Assign `~/.nujo` to a separate variable as a constant and use it instead.